modules/ROOT/pages/push/proc_preparing-ups-for-use-with-ios.adoc

```diff
@@ -0,0 +1,107 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: proc_preparing-ups-for-use-with-ios
+
+[id='{context}_preparing-ups-for-use-with-ios']
+= Preparing {unifiedpush-service} For Use With iOS
+
+Before using {unifiedpush-service} with iOS, first you need to submit a request for a new digital certificate, which is based on a public/private key. The Certificate itself is acting as the public key, and when you request it, a private key is added to your KeyChain tool. The Cerficate will be used later on, to request an SSL certificate for the Apple Push Network Service, which will allow the AeroGear Push Server to send notification messages to it.
```

Comment:
@jhellar does " The Cerficate will be used later on, to request an SSL certificate for the Apple Push Network Service, which will allow the AeroGear Push Server to send notification messages to it." meet your ask (in rocket chat)?